tests/app/views/contexts/test_submission_metatdata_context.py

```diff
@@ -28,6 +29,21 @@ def test_metadata_survey_type_social(app: Flask):
         }
 
 
+def test_metadata_survey_type_health(app: Flask):
```

Comment:
This is identical to one about bar the theme, it should be parametrised.

Reply:
Done. Thanks.